packaging/common/cfengine-hub/postinstall.sh

```diff
@@ -274,7 +274,7 @@ do
   if [ -f /usr/share/man/man8/$i.8.gz ]; then
     rm -f /usr/share/man/man8/$i.8.gz
   fi
-  $PREFIX/bin/$i -M > /usr/share/man/man8/$i.8 && gzip /usr/share/man/man8/$i.8
+  $PREFIX/bin/$i -M > /usr/share/man/man8/$i.8 && gzip /usr/share/man/man8/$i.8 || true
```

Comment:
Humm, what about || echo "Install gzip and run `$i -M > /usr/share/man/man8/$i.8`

Reply:
I don't think it's necessary. On the only platform I've found recently to not contain `gzip`, `man` was accepting uncompressed pages no problem (although other man pages were shipped compressed, inside package) (also worth noting that `man` wasn't installed by default - I had to install it myself).

Reply:
You could split this up and check that gzip is available.
```suggestion
  $PREFIX/bin/$i -M > /usr/share/man/man8/$i.8
  command -v gzip 2>/dev/null && gzip /usr/share/man/man8/$i.8
```

Might make a bit more sense pipeline wise. I know shellcheck doesn't like you using `&& ||` due to slightly undefined/imprecise behavior like if `$PREFIX/bin/$i` fails then `true` is still called which isn't really what we want right?

Reply:
> if `$PREFIX/bin/$i` fails then `true` is still called which isn't really what we want right?

Hmm, old code was calling `gzip` only if `$PREFIX/bin/$i -M` succeeded. Maybe we want to preserve this behaviour, too?